RELEASE_NOTES.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## 1.8.1
+### Features
+- Many networking-related flags have been added to `podman pod create` to enable customization of pod networks, including `--add-host`, `--dns`, `--dns-opt`, `--dns-search`, `--ip`, `--mac-address`, `--network`, and `--no-hosts`
+- The `podman ps --format=json` command now includes the ID of the image containers were created with
```

Comment:
'with' is generally a joining word and especially odd to end a "sentence" with that doesn't have periods.  Perhaps "image containers" -> "image that the containers"